Libs/DICOM/Widgets/ctkDICOMServerNodeWidget2.cpp

```diff
@@ -1466,7 +1498,7 @@ void ctkDICOMServerNodeWidget2::readSettings()
     // a dummy example
     QMap<QString, QVariant> defaultServerNode;
     defaultServerNode["Name"] = QString("ExampleHost");
-    defaultServerNode["Verification"] = tr("unknown");
+    defaultServerNode["Verification"] = ctkDICOMServerNodeWidget2::tr("unknown");
```

Comment:
In `ctkDICOMServerNodeWidget2` member functions it should not be necessary to add `ctkDICOMServerNodeWidget2::`